README.md

```diff
@@ -53,41 +80,50 @@ When you ask your assistant to “Turn on the light”, it will use the auth bea
     "automation" :
     {
      "name": "automation",
-     "url": "https://YOUR-URL/openhabGoogleAssistant"
+     "url": "https://us-central1-YOUR-APPLICATION.cloudfunctions.net/openhabGoogleAssistant"
     }
   }
 }
 ```
 
-* Afterwards upload this action file using the following command:
+* Afterwards deploy this action file using the following command:
 ```
-gactions update --action_package action.json --project <YOUR-GOOGLE-CLOUD-PROJECT-ID>
+gactions update --action_package action.json --project <YOUR-APPLICATION>
 ```
 
-Google Assistant will call the service endpoint: `https://YOUR-URL/openhabGoogleAssistant`.
+Google Assistant will call the service endpoint: `"https://us-central1-YOUR-APPLICATION.cloudfunctions.net/openhabGoogleAssistant"`.
 This web service will receive parameters (intents) from Google and will query/modify openHAB items through openHAB-cloud depending on those parameters.
 
 * You need to Add "App information”, including name and account linking details to the Actions Console
 * Afterwards please run the following command in the gaction CLI:
 ```
-gactions test --action_package PACKAGE_NAME --project AGENT_ID
+gactions test --action_package action.json --project <YOUR-APPLICATION>
 ```
 
-### Account Linkage & OAuth2:
-
-To enable the OAuth account linkage you need to setup the according values in the Actions Console: 
-
-Set up account linking:
-
-* Grant type: Autorization code
-* Client ID: `<YOUR-ID>`
-* Client secret: `<YOUR-SECRET>`
-* Authorization URL: `https://<YOUR-URL>/oauth`
-* Token URL: `https://<YOUR-URL>/token`
+## Setup your Database
+* SSH into to your openhab cloud instance
+* Open the mongodb client `mongo` and enter these commands
+```
+use openhab
+db.oauth2clients.insert({ clientId: “client id here”, clientSecret: “secret here”})
```

Comment:
Please use the same way how we descrive things that need to be filled in e.g. <YOUR-SECRET> etc. Dont introduce another way..people might oversee it.

Reply:
sure, i just copied this from the long thread on the forums.